packages/di/src/decorators/inject.decorator.ts

```diff
@@ -0,0 +1,19 @@
+import 'reflect-metadata';
```

Comment:
We should check if we can have this statement once in the index file of the library so that we don't repeat it in other files.